tutorials/enricher/pom.xml

```diff
@@ -13,9 +13,9 @@
 
   <properties>
     <!-- change this if you desire to use a different version of Apex Core -->
-    <apex.version>3.4.0</apex.version>
+    <apex.version>3.6.0-dt20161215</apex.version>
```

Comment:
@amberarrow  this is for SPOI-10583. The goal was to validate the examples with the latest apex-core, apex-malhar versions released by DT in RTS and not the open-source ones.